lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -44,11 +44,12 @@ class DOM {
 
   /**
    * @param {string} selector
+   * @param {Element=} context Parent node to query for templates within.
    * @return {!DocumentFragment} A clone of the template content.
    * @throws {Error}
    */
-  cloneTemplate(selector) {
-    const template = this._document.querySelector(selector);
+  cloneTemplate(selector, context = this._document) {
```

Comment:
are you comfortable with using `this.templatesContext` instead? 

```js
cloneTemplate(selector) {
  const template = this.templatesContext.querySelector(selector);
```

this would be ideal for me. I'd set it right after instantiating the instance. 
it would default to `this._document`